reV/SAM/generation.py

```diff
@@ -371,128 +436,49 @@ def reV_run(cls, points_control, res_file, site_df,
 
 
 class Solar(Generation, ABC):
```

Comment:
Same here.. I keep thinking that these are user classes not base/abstact classes

Reply:
agreed